dev-docs/bidders/admatic.md

```diff
@@ -6,10 +6,11 @@ pbjs: true
 pbs: false
 biddercode: admatic
 media_types: banner,video,native
-tcfeu_supported: false
-usp_supported: false
-coppa_supported: false
-schain_supported: false
+tcfeu_supported: true
+usp_supported: true
+coppa_supported: true
+gpp_supported: true
```

Comment:
```suggestion
gpp_sids: tcfeu, tcfca, usnat, usstate_all, usp
```

`gpp_supported` is deprecated and superseded by `gpp_sids` . Add the support section ids your adapter supports

Reply:
@fatihkaya84 can you fix this? Then this is ready to merge :smile:

Reply:
> gpp_sids: tcfeu, tcfca, usnat, usstate_all, usp

hi @muuki88 sorry late reply :(
thank you very much for your interest and understanding

Reply:
No worries. We are all working very async :smile: